### PR TITLE
Release v0.12.0

### PR DESCRIPTION
**Pull request by dewjam**
**Description**
Karpenter v0.12.0 release

**How was this change tested?**
Ran `make install` and tested scale up/down.

**Does this change impact docs?**
- [ ] Yes, PR includes docs updates
- [ ] Yes, issue opened: # 
- [x] No

**Release Note**

```release-note

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by njtran** [comment]
Screen Shot 2022-06-21 at 2 42 05 PM
The UserData docs aren't showing up in the drop down for me, but I can still click on it. Is this happening for you?

**Comment by njtran** [comment]
The UserData docs aren't showing up in the drop down for me, but I can still click on it. Is this happening for you?
Screen Shot 2022-06-21 at 2 42 05 PM

**Comment by njtran** [comment]
The UserData docs aren't showing up in the drop down for me, but I can still click on it. Is this happening for you?

File: website/content/en/v0.12.0/AWS/user-data.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
+This document describes how you can customize the UserData that will be specified on your EC2 worker nodes, without using a launch template.
```
Comment (njtran) [inline website/content/en/v0.12.0/AWS/user-data.md:1]: This needs something like this

```
---
title: "User Data"
linkTitle: "User Data"
weight: 90
description: >
  Learn how Karpenter enables User Data
---
```

Comment (dewjam) [reply website/content/en/v0.12.0/AWS/user-data.md:1]: Yeah, this was missing from previous website versions as well.  I can certainly add this.

Comment (dewjam) [reply website/content/en/v0.12.0/AWS/user-data.md:1]: Looks like we did that on purpose:  https://github.com/aws/karpenter/commit/2873859fd8074e04969a60e347593dc83dc60172

Comment (dewjam) [reply website/content/en/v0.12.0/AWS/user-data.md]: Removed the user-data.md page for all but the preview site.

**Review by spring1843 — APPROVED** [review]

**Pull request merged**